src/EnergyPlus/OutputProcessor.cc

```diff
@@ -2467,6 +2467,10 @@ namespace OutputProcessor {
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         auto &op(state.dataOutputProcessor);
 
+        if (!op->MeterValue.allocated()) {
+            return;
+        }
```

Comment:
@Myoldmopar note that this new code protects this call before meters are allocated. Similar band-aid since each of the loops below are from 1 to op->NumEnergyMeters. I guess it's possible to know the number of meters before those meters are actually set up. Maybe the new calls (those that change the calling order) could be delayed until something like state.dataGlobal->MetersHaveBeenInitialized = true ? If that works then could unload those meter functions (i.e., remove all new protections) that are called throughout the simulation.